src/utils/renderChainPath.ts

```diff
@@ -1,8 +1,31 @@
 /**
- * Method to render a safe, chained path notation
+ * Method to render a safe, optional chained path notation
  * @param path the path definition (example: website[0].url)
+ * @param legacyMode force Portman to render the full path
  */
-export const renderChainPath = (path: string): string => {
+export const renderChainPath = (path: string, legacyMode = false): string => {
```

Comment:
@nicklloyd is there an easy way to get the cli Options in this utility? without going through all parent functions and passing them as optional parameters?

Reply:
only if you add this method onto Portman itself (or higher up the chain)...